recipe/meta.yaml

```diff
@@ -1,24 +1,32 @@
+{% set version = "3.6.0" %}
+
 package:
   name: python
-  version: 3.5.2
+  version: {{ version }}
 
 source:
-  fn: Python-3.5.2.tgz
-  url: https://www.python.org/ftp/python/3.5.2/Python-3.5.2.tgz
-  sha256: 1524b840e42cf3b909e8f8df67c1724012c7dc7f9d076d4feef2d3eff031e8a0
+  fn: Python-{{ version }}.tgz
+  url: https://www.python.org/ftp/python/{{ version }}/Python-{{ version }}.tgz
+  sha256: aa472515800d25a3739833f76ca3735d9f4b2fe77c3cb21f69275e0cce30cb2b
   patches:
       - omit-local-site-packages.patch  # [osx]
       - osx64-dist.patch                # [osx and x86_64]
       - win-find_exe.patch              # [win]
       - win-library_bin.patch           # [win]
+      # Adjust FD_SETSIZE to allow select() to wait on more connections
+      # simultaneously.  Desirable for serious use of Tornado on Windows.
+      # See also:
+      # - https://bugs.python.org/issue28708
+      # - https://github.com/zeromq/libzmq/issues/1165
+      - fd_setsize.patch                # [win]
```

Comment:
Maybe we should be taking this on separately. Namely as a PR ( https://github.com/conda-forge/python-feedstock/pull/87 ). Doesn't feel like the sort of thing we should bury in a release.